### ISSUE TITLE
ImportError: cannot import name 'get_interface_codes' from 'vyper.cli.vyper_compile' 

### ISSUE THREAD

**Issue by PatrickAlphaC** [vyperlang/titanoboa#176]
Using the new `0.4.0b1+commit.6fb750a` edition of vyper, the following code breaks:

```python
import boa
```

then try to run the script with `python script_name.py`

Error:
```error
Traceback (most recent call last):
  File "/xxx/script.py", line 1, in <module>
    import boa
  File "/xxx/site-packages/boa/__init__.py", line 8, in <module>
    from boa.interpret import BoaError, load, load_abi, load_partial, loads, loads_abi, loads_partial
  File "/xxx/site-packages/boa/interpret.py", line 6, in <module>
    from vyper.cli.vyper_compile import get_interface_codes
ImportError: cannot import name 'get_interface_codes' from 'vyper.cli.vyper_compile' (/xxx/site-packages/vyper/cli/vyper_compile.py)
```

boa version: 0.1.8

**Comment by PatrickAlphaC** [comment]
After trying by installing the `vyper-0.4.0` branch of titanoboa, I get this error now:

```
Traceback (most recent call last):
  File "/xxx/hi.py", line 1, in <module>
    import boa
  File "/xxx/venv/lib/python3.10/site-packages/boa/__init__.py", line 5, in <module>
    from boa.contracts.vyper.vyper_contract import check_boa_error_matches
  File "/xxx/venv/lib/python3.10/site-packages/boa/contracts/vyper/vyper_contract.py", line 45, in <module>
    from boa.contracts.vyper.compiler_utils import (
  File "/xxx/venv/lib/python3.10/site-packages/boa/contracts/vyper/compiler_utils.py", line 13, in <module>
    from vyper.codegen.module import _globally_reachable_functions
ImportError: cannot import name '_globally_reachable_functions' from 'vyper.codegen.module' (/xxx/venv/lib/python3.10/site-packages/vyper/codegen/module.py)
```

**Comment by charles-cooper** [comment]
should be fixed on latest vyper-0.4.0
